expand/expand_spec.go

```diff
@@ -309,6 +310,15 @@ func getAzControlPlaneConfig() *corev1beta1.ProviderConfig {
 	return &obj
 }
 
+func getAwsControlPlaneConfig() *corev1beta1.ProviderConfig {
```

Comment:
small change please remove getXXXX. Make it only XXXX. As mentioned here: https://golang.org/doc/effective_go.html#Getters

Reply:
thanks! i've changed function name and function reference